submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,197 @@
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/contacts.png" alt="" >
+							</div>
+							<p>Contact</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/tanslation.png" alt="" >
+							</div>
+							<p>Translate</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/videos.png" alt="" >
+							</div>
+							<p>Video</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/search.png" alt="" >
+							</div>
+							<p>Search</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/photos.png" alt="" >
+							</div>
+							<p>Photo</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/apps.png" alt="" ></div>
+							<p>Apps</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/documents.png" alt="" >
+							</div>
+							<p>Document</p>
+						</div></a
+					>
+				</div>
+
+				<input 
+					class="second-input"
+					type="checkbox"
+					id="second1-navbar-button"
+				>
+				<label for="second1-navbar-button" class="second-navbar-button">
+					<div class="buttom-poup-app"><p>More</p></div>
+				</label>
+
+				<div class="line"></div>
+
+				<div class="app-second">
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/chat.png" alt="" ></div>
+							<p>Calendar</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/classroom.png" alt="" >
+							</div>
+							<p>Mail</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/collection.png" alt="" >
+							</div>
+							<p>News</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/mars.png" alt="" ></div>
+							<p>Map</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/blogs.png" alt="" ></div>
+							<p>Files</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/promo.png" alt="" ></div>
+							<p>Contact</p>
+						</div></a
+					>
+
+					<p class="text-end">Even more from Cattans</p>
```

Comment:
It must be a link. Also, add some space to the bottom 

![image](https://user-images.githubusercontent.com/16196199/99585184-a741e180-29ee-11eb-82e0-c309da1bf14e.png)